### PR TITLE
a few more tools for the wizard

### PR DESCRIPTION
**Pull request by ericxtang**

### CODE REVIEW

File: cmd/livepeer_cli/wizard_deposit.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func (w *wizard) deposit() {
+	fmt.Printf("Current deposit: %v\n", w.getDeposit())
+	fmt.Printf("Current token balance: %v\n", w.getTokenBalance())
+	fmt.Printf("Enter Deposit Amount - ")
+	amount := w.readInt()
+	httpGet(fmt.Sprintf("http://%v:%v/deposit?amount=%v", w.host, w.httpPort, amount))
```
Comment (yondonfu) [inline cmd/livepeer_cli/wizard_deposit.go]: These should be POST requests right? Don't think we necessarily need to switch over now if a lot of the state changing end points already use GET requests, but we should probably come back and do so later especially if the HTTP server is used outside of the CLI wizard

**Review by yondonfu — APPROVED** [review]
LGTM 🚢  Left a comment about GET vs POST requests for the web server, but that doesn't have to be addressed immediately

File: cmd/livepeer_cli/wizard.go [hunk 2]
```diff
@@ -45,3 +46,24 @@ func (w *wizard) readDefaultString(def string) string {
...
+	for {
+		fmt.Printf("> ")
+		text, err := w.in.ReadString('\n')
+		if err != nil {
+			log.Crit("Failed to read user input", "err", err)
+		}
+		if text = strings.TrimSpace(text); text == "" {
+			continue
+		}
+		val, err := strconv.Atoi(strings.TrimSpace(text))
```
Comment (dob) [inline cmd/livepeer_cli/wizard.go:62]: why do you have to TrimSpace again?

**Review by dob — COMMENTED** [review]
Yah I agree with Yondon about state changing requests should be POSTs if possible.

File: cmd/livepeer_cli/wizard.go (hunk 2 above)
Comment (ericxtang) [reply cmd/livepeer_cli/wizard.go:62]: I think otherwise it errors out... Copied from puppeth

**Pull request merged**